usbsdmux/service.py

```diff
@@ -149,7 +156,10 @@ def main():
     while True:
         try:
             conn, addr = sock.accept()
-            answer = process_request(conn.recv(4096).decode())
+            answer = process_request(
+                conn.recv(4096).decode(),
+                not self.force
```

Comment:
`self` is not defined here.

Reply:
Ooops. I should have tested the service after adding this change. I've fixed this error and did at least some basic testing now.